src/virtual-machine.js

```diff
@@ -295,6 +296,25 @@ class VirtualMachine extends EventEmitter {
                 if (error) return reject(error);
                 resolve(res);
             });
+        })
+        .catch(error => {
+            try {
+                const sb1 = new SB1File(input);
+                const json = sb1.json;
+                json.projectVersion = 2;
+                return [json, sb1.zip];
```

Comment:
I assume this line needs to be a resolve because this is all still happening as part of a Promise...